src/components/app-button-menu/AppButtonMenu.tsx

```diff
@@ -65,12 +64,13 @@ const AppButtonMenu = <T extends Pick<CategoryNameInterface, '_id'>>({
     setSelectedItems([])
   }
 
-  const { loading, response } = useAxios<T[]>({
-    service,
-    defaultResponse: defaultResponses.array
+  const { data: response, isLoading } = useQuery({
+    queryKey: [title],
+    queryFn: service
   })
 
   const filteredItems = useMemo(() => {
+    if (!response) return []
```

Comment:
Add a braces for consistancy with our codebase
```suggestion
    if (!response) {
    return []
    }
```